helm/dscp-api/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: dscp-node
   repository: https://digicatapult.github.io/dscp-node/
-  version: 3.8.0
+  version: 4.0.0
 - name: dscp-ipfs
   repository: https://digicatapult.github.io/dscp-ipfs/
   version: 2.2.0
```

Comment:
Need to upgrade to as yet unreleased version of `dscp-ipfs`

Reply:
just waiting for https://github.com/digicatapult/dscp-ipfs/actions/runs/2657636749 to finish